ABI/main-contract.js

```diff
@@ -1,308 +1,308 @@
 const mainContractABI = [
```

Comment:
All ABI and other constants need to be in a directory called constants.

Reply:
I have done all the directory related changes in another pull request.
https://github.com/inbloxme/inblox-handlename-package/pull/8